scripts/misc/generate_vep_examples.pl

```diff
@@ -110,8 +110,11 @@ =head1 CONTACT
     my $name;
     my $sth = $real_vfa->db->dbc->prepare(qq{
       SELECT variation_name
-      FROM variation_feature
+      FROM variation_feature vf
+        LEFT JOIN failed_variation fv
+        ON vf.variation_id = fv.variation_id
       WHERE consequence_types LIKE ?
+      AND fv.variation_id IS NULL
       LIMIT 1
```

Comment:
Should this query be deterministic? If yes should include an order by on variation_name
https://dev.mysql.com/doc/refman/5.6/en/limit-optimization.html

Reply:
Thanks for checking @helensch. Currently I get the same examples if the same database version is picked. I don't think there is a need of enforcing the sql to be deterministic. The generated examples are not particularly interesting besides being one a missense, one an intron variant and one a frameshift.

Reply:
If you want the same examples with a LIMIT you should use an order by. Getting the same examples at the moment is probably due to the index on name - just something to watch for in future if using LIMIT

Reply:
Thanks @helensch, that's a good point! Will keep a note about it.